packages/cubejs-cli/src/config.js

```diff
@@ -77,6 +89,22 @@ export class Config {
       return config;
     }
 
+    const answer = await this.cloudTokenReq({
+      url: `${process.env.CUBE_CLOUD_HOST || 'https://cubecloud.dev'}/v1/token`,
+      method: 'POST',
+      body: {
+        token: authToken
+      }
+    });
+    
+    if (answer.error) {
```

Comment:
If `cloudTokenReq` is a general method for calling all available API, I think we should move this check for error to it.